### PR TITLE
[AIDAPP-172]: Introduce Service Requests report for the Report Library

### PR DESCRIPTION
**Pull request by ketan-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/AIDAPP-172

### Technical Description

Added service request widget as required.

### Any deployment steps required?

Run "composer ci" command because we added new module report.

### Are any Feature Flags Added?

No

_______________________________________________

### CODE REVIEW

File: app-modules/report/src/Registries/ReportRbacRegistry.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+        $roleRegistry->registerApiRoles(
+            module: '',
+            path: ''
+        );
+
+        $roleRegistry->registerWebRoles(
+            module: '',
+            path: ''
+        );
```
Comment (Orrison) [inline app-modules/report/src/Registries/ReportRbacRegistry.php:56]: I am pretty sure this is what is causing the test failures in CI

```suggestion
        $roleRegistry->registerApiRoles(
            module: 'report',
            path: 'roles/api'
        );

        $roleRegistry->registerWebRoles(
            module: 'report',
            path: 'roles/web'
        );
```

**Review by Orrison — CHANGES_REQUESTED** [review]

File: app-modules/report/src/Registries/ReportRbacRegistry.php (hunk 1 above)
Comment (ketan-canyon) [reply app-modules/report/src/Registries/ReportRbacRegistry.php]: Done.
Error resolved. thanks.

File: app-modules/report/config/roles/api/report_roles.php
Comment (Orrison) [inline app-modules/report/config/roles/api/report_roles.php:1]: Let's delete this file for now as we don't want to create any roles at the moment for this.

We can add a `.gitkeep` in it's place to keep the folder structure here though.

File: app-modules/report/config/roles/web/report_roles.php
Comment (Orrison) [inline app-modules/report/config/roles/web/report_roles.php:1]: Let's delete this file for now as we don't want to create any roles at the moment for this.

We can add a `.gitkeep` in it's place to keep the folder structure here though.

**Review by Orrison — CHANGES_REQUESTED** [review]

File: app-modules/report/config/roles/api/report_roles.php
Comment (ketan-canyon) [reply app-modules/report/config/roles/api/report_roles.php:1]: Done.

File: app-modules/report/config/roles/web/report_roles.php
Comment (ketan-canyon) [reply app-modules/report/config/roles/web/report_roles.php:1]: Done

**Review by Orrison — APPROVED** [review]

**Pull request merged**